### PR TITLE
Now we can build on Yosemite 

### PR DESCRIPTION
**Pull request by doutriaux1**
@aashish24 @dlonie @jbeezley  @chaosphere2112 please test and review

### CODE REVIEW

File: CMake/cdat_modules/vtk_external.cmake [hunk 1]
```diff
@@ -26,21 +26,21 @@ list(APPEND vtk_build_args
 )
 
 # Use cdat zlib
-if(NOT CDAT_USE_SYSTEM_ZLIB)
-  list(APPEND vtk_build_args
-    -DZLIB_INCLUDE_DIR:PATH=${cdat_EXTERNALS}/include
-    -DZLIB_LIBRARY:FILEPATH=${cdat_EXTERNALS}/lib/libz${_LINK_LIBRARY_SUFFIX}
-  )
-endif()
+#if(NOT CDAT_USE_SYSTEM_ZLIB)
```
Comment (dlonie) [inline CMake/cdat_modules/vtk_external.cmake]: Aashish will comment more, but is there a compelling reason to make these changes in master?

Comment (doutriaux1) [inline CMake/cdat_modules/vtk_external.cmake]: yes I turned off zlib and libxml2 libxslt etc.. there are on every system. I guess we can uncomment if you want but I'm not positive about what happen to CDAT_USE_SYSTEM_ZLIB now that we don't have include(libz_pkg)

Comment (aashish24) [inline CMake/cdat_modules/vtk_external.cmake]: I have to review bit more but basically if we use system PNG, JPEG etc, then we have to pass the appropriate flags to VTK or else VTK will build its on vtkjpeg, and vtkpng as well. So thing will build but won't run. 

Also, I don't know why we took out cdms?

**Comment by doutriaux1** [comment]
@aashish24 , I did the VTK_USE_SYSTEM_[lib] as you recommended

**Comment by doutriaux1** [comment]
@aashish24 if you're uncomfortable with all this just put it back as before, don't comment out the files in CMakeLists.txt and uncomment the lines in VTK. I'm ok with it as well, just test again on your mac if you do so please.

**Comment by doutriaux1** [comment]
@aashish24 wait a bit just found a possible issue. Don't merge yet

**Comment by doutriaux1** [comment]
@aashish24 ok good to go. Pass even with CDAT_BUILD_MODE=ALL on RH6

File: CMake/cdat_modules/libcdms_deps.cmake [hunk 2]
```diff
@@ -0,0 +1,3 @@
+set(libcdms_deps ${netcdf_pkg} 
+              ${jasper_pkg} ${g2clib_pkg} ${tiff_pkg}
+              ${png_pkg} ${jpeg_pkg} )
```
Comment (aashish24) [inline CMake/cdat_modules/libcdms_deps.cmake]: why depending on png? and jpeg?

Comment (doutriaux1) [inline CMake/cdat_modules/libcdms_deps.cmake]: jasper needs jpeg 
g2clib needs png

File: CMake/cdat_modules/vtk_external.cmake (hunk 1 above)
Comment (doutriaux1) [inline CMake/cdat_modules/vtk_external.cmake]: ok i fixed that already. And libcdms was simply moved out to its own repo, easier to maintain

**Comment by doutriaux1** [comment]
awesome! thanks!

**Pull request merged**